dev-docs/Cloud-Env-Testing.md

```diff
@@ -0,0 +1,117 @@
+# Cloud Environment Testing
+
+The [Create Environment](https://github.com/elastic/cloudbeat/actions/workflows/test-environment.yml) GitHub action
+deploys a full-featured cloud environment, pre-configured with all our integrations. It also includes features for
+running sanity testing and automated deletion.
+
+## How to Run the Workflow
+
+Follow these steps to run the workflow:
+
+1. Go to [`Actions > Create Environment`](https://github.com/elastic/cloudbeat/actions/workflows/test-environment.yml).
+
+   ![Navigate to Actions](https://github.com/elastic/cloudbeat/assets/99176494/2686668f-7be6-4b55-a37b-e37426c1a0e1)
+
+2. Click the `Run workflow` button.
+
+   ![Run Workflow](https://github.com/elastic/cloudbeat/assets/99176494/115fdd53-cff7-406a-bc3d-d65d5199389f)
+
+3. Complete the required input fields:
+
+    - **`deployment_name`**: Name your environment (Allowed characters: a-zA-Z0-9 and `-`). For
+      instance: `john-8-7-2-June01`.
+
+    - **`ec-api-key`**: Elastic Cloud API Key. Refer to
+      the [Cloud API Keys](https://www.elastic.co/guide/en/cloud/current/ec-api-authentication.html) documentation to
+      generate your token.
+
+    - **`elk-stack-version`**: Specify the version of Elastic Cloud stack, either a SNAPSHOT or a build candidate (BC)
+      version. The default value is `8.8.0`. Check the available
+      versions [here](https://artifacts-staging.elastic.co/dra-info/index.html).
+
+    - **`ess-region`**: Indicate the Elastic Cloud deployment region. The default value is `gcp-us-west2`, which
+      supports
+      snapshot and build candidate (BC) versions. Specify a different region only if necessary.
+
+   ![Enter Inputs](https://github.com/elastic/cloudbeat/assets/99176494/06d8144d-13cc-4e13-92fc-19f52ce8206b)
+
+4. Optionally, modify other input values if required:
+
+    - `docker-image-override` (optional): Use this to replace the default Docker image for build candidate (BC)
```

Comment:
i would add a short explanation regarding how to get the correct build hash and a link to the [DRA](https://artifacts-staging.elastic.co/dra-info/index.html).

Reply:
I provided a link + explanation (not to the DRA since it's not there, unless I miss something)

Reply:
go to DRA, you can select any version from staging on snap shop, click on JSON report. you will be provided the url to the desired stack release

![image](https://github.com/elastic/cloudbeat/assets/51442161/ba030d35-de44-4394-9669-f7785f09971e)

![image](https://github.com/elastic/cloudbeat/assets/51442161/4414b1d5-f370-4824-9284-17c21c062ed1)

![image](https://github.com/elastic/cloudbeat/assets/51442161/f06245fd-aab4-4961-a6bb-bbb38c82afe5)